# error_handling.py
class ProductError(Exception):
    pass

class AuthenticationError(Exception):
    pass
